app.rb

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env ruby
```

Comment:
This allows the terminal to know what interpreter to use to run this script. So instead of...

```
ruby app.rb
```

You can now do

```
./app.rb
```